app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeriesFragment.java

```diff
@@ -74,10 +87,27 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        View v = inflater.inflate(R.layout.fragment_events, container, false);
-        ButterKnife.inject(this, v);
-        return v;
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        if (getArguments() != null && getArguments().getBoolean(ARGS_IS_PORTRAIT, false)) {
+            View header = getLayoutInflater(null)
```

Comment:
encoding orientation looks not right. It should be handled in onConfigurationChanged.